src/main/java/com/amazon/ion/impl/IonRawTextWriter_1_1.kt

```diff
@@ -401,7 +401,7 @@ class IonRawTextWriter_1_1 internal constructor(
     override fun stepInExpressionGroup(usingLengthPrefix: Boolean) {
         confirm(numAnnotations == 0) { "Cannot annotate an expression group" }
         confirm(currentContainer == EExpression) { "Can only create an expression group in a macro invocation" }
-        openValue { output.appendAscii("(:") }
+        openValue { output.appendAscii("(::") }
```

Comment:
The drive-by syntax fix. Tests fixed in `IonManagedWriter_1_1_Test` and `IonRawTextWriterTest_1_1`.